customer_satisfaction/steps/model_train.py

```diff
@@ -0,0 +1,39 @@
+import logging
+from zenml.steps import step, Output
+from model.model_dev import ModelTraining
+import pandas as pd
+
+# you can uncomment the following two lines if you're changing the model 
+# from sklearn.ensemble import RandomForestRegressor
+# from xgboost import XGBRegressor
+from lightgbm import LGBMRegressor

```

Comment:
Maybe no need to implement this, but wondering if there's a more elegant way to implement this functionality? Maybe args passed in at the command line, or toggling this somewhere else in the code? Feels a bit strange to have to uncomment lines in the code to run different models.

Reply:
I can try implementing this.

Reply:
yep, I implemented this, basically I made a config file and one can change config files to switch between models.